securedrop-admin

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python2
+"""
+SecureDrop Admin Toolkit.
+
+For use by administrators to install, maintain, and manage their SD
+instances.
+"""
+
+import argparse
+import subprocess
+import os
+import sys
+import getpass
+import logging
+
+sdlog = logging.getLogger('SD')
+
+SD_DIR = os.path.dirname(os.path.realpath(__file__))
+ANSIBLE_PATH = os.path.join(SD_DIR, "./install_files/ansible-base")
+SITE_CONFIG = os.path.join(ANSIBLE_PATH, "group_vars/all/site-specific")
+VENV_DIR = os.path.join(SD_DIR, ".venv")
+VENV_ACTIVATION = os.path.join(VENV_DIR, 'bin/activate_this.py')
+
+
+def logger_setup(verbose=False):
+    """ Configure logging handler """
+    # Set default level on parent
+    sdlog.setLevel(logging.DEBUG)
+    level = logging.DEBUG if verbose else logging.INFO
+
+    stdout = logging.StreamHandler(sys.stdout)
```

Comment:
I mentioned that you could send logger output to `sys.stdout` earlier, but that's not necessarily the best place to send it. It seems like ideally we'd have errors go to `sys.stderr` and informative messages go to `sys.stdout`. There are of course many ways of doing that. One way would be to configure two loggers and use one for info messages and another for error messages. Personally, I think it would be more simple + clean to just use `print` for info messages and change the logger output to go to `sys.stderr`.